### PR TITLE
Switches linting to eslint & removes jshint and jscs

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

**Comment by FagnerMartinsBrack** [comment]
Good stuff. What's the main reason for the change, though?

**Comment by realityking** [comment]
Sorry, should have given more context. jscs is unmaintained - the repo is archived - and this merges two configs into one, making things a bit easier to grasp.

**Comment by FagnerMartinsBrack** [comment]
I see this looks good then assuming that the behavior remains unchanged.

File: package.json [hunk 1]
```diff
@@ -35,12 +35,11 @@
     "grunt": "1.0.2",
     "grunt-compare-size": "0.4.2",
     "grunt-contrib-connect": "1.0.2",
-    "grunt-contrib-jshint": "1.1.0",
     "grunt-contrib-nodeunit": "2.0.0",
     "grunt-contrib-qunit": "2.0.0",
     "grunt-contrib-uglify": "2.3.0",
     "grunt-contrib-watch": "1.1.0",
-    "grunt-jscs": "3.0.1",
+    "grunt-eslint": "^20.2.0",
```
Comment (FagnerMartinsBrack) [inline package.json]: Just a small comment, can we use fixed versions here?

Comment (realityking) [reply package.json]: Sorry, hadn't noticed you're using fixed versions for dependencies. Fixed now.

**Comment by realityking** [comment]
Behaviour should be the same, I used https://github.com/brenolf/polyjuice for the conversion + couple of manual fixes for the cases it didn't cover.

**Pull request merged**